app/uk/gov/hmrc/helptosavefrontend/util/Toggles.scala

```diff
@@ -25,10 +25,10 @@ object Toggles {
   case class FEATURE[A](name: String, conf: Configuration, unconfiguredVal: Option[A], logger: Logger) {
     def enabled(): FEATURE_THEN[A] = {
       conf.getBoolean(s"feature-toggles.$name.enabled") match {
-        case Some(b) =>
+        case Some(b) ⇒
           logger.info("FEATURE: " + name + (if (b) " enabled" else " disabled"))
           FEATURE_THEN(name, b, unconfiguredVal, logger)
-        case None => throw new RuntimeException(s"FEATURE($name) is not present in configuration file - misconfigured")
+        case None ⇒ throw new RuntimeException(s"FEATURE($name) is not present in configuration file - misconfigured")
```

Comment:
here => are not aligned again for some reason

Reply:
the alignment only happens for single line case statements (https://github.com/scala-ide/scalariform#alignsinglelinecasestatements) - the `Some` branch has two lines